cocos/2d/framework/ui-renderer.ts

```diff
@@ -198,7 +198,10 @@ export class UIRenderer extends Renderer {
         }
     }
 
-    protected _renderData: RenderData | null = null;
+    /**
+     * @engineInternal NOTE: this is engine internal only, we need to access `_renderData` for we don't wan't to see the deprecated warning.
```

Comment:
`don't wan't to` -> `don't want to`

Reply:
updated https://github.com/cocos/cocos-engine/pull/14572/commits/467e70ef6c4d557169328ba54be7f90f0f59386e